Polls/ViewControllers/QuestionListViewController.swift

```diff
@@ -26,6 +26,20 @@ class QuestionListViewController : UITableViewController, QuestionDetailViewCont
     loadData()
   }
 
+  override func viewWillAppear(animated: Bool) {
+    super.viewWillAppear(animated)
+
+    if let selectedIndex = tableView.indexPathForSelectedRow() {
```

Comment:
I feel the little tricks like these distinguish iOS masters from people that developed an iOS app..